### PR TITLE
pkg/route: update owners

### PR DESCRIPTION
**Pull request by alebedev87**
Remove inactive, add current teams members.

### CODE REVIEW

File: pkg/route/OWNERS [hunk 1]
```diff
@@ -2,15 +2,11 @@ approvers:
   - frobware
   - knobunc
   - Miciah
-  - sgreene570
-  - smarterclayton
```
Comment (Miciah) [inline pkg/route/OWNERS:6]: You could move these to `emeritus_approvers` per <https://www.kubernetes.dev/docs/guide/owners/#cleanup>.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Comment by bertinatto** [comment]
/assign @Miciah 
for approval

File: pkg/route/OWNERS (hunk 1 above)
Comment (alebedev87) [reply pkg/route/OWNERS:6]: Clayton added to the emeritus approvers for his great contribution to all things OpenShift. The others are nice guys but I don't see a reason to keep them in OWNERS.

**Pull request merged**